README.md

```diff
@@ -1,10 +1,41 @@
-# Space Shooter game (HTML5, Canvas, Vanilla JS)
-![Poster](poster.jpg)
+## Space Shooter
+The game created using `HTML5` `Canvas` `Vanilla JS`.
+
+## Demo
+Click the on the poster and to see a demo
+[![Space Shooter Demo - Youtube](./posters/poster.jpg)](https://youtu.be/MTrX7byllUQ)
+
+## Screenshots
+| ![Poster 1](./posters/poster-2.jpg) | ![Poster 2](./posters/poster-3.jpg) |
+|---------------------------------------------|---------------------------------------------|
+
+## Game Features
+
+- Health bar for the rocket
+- Score for check your achievements
+- Theme music and custom sounds
+- Sprite animation
+   - asteroid flying
+   - bullet shoots
+   - rocket fire burning
+- 6 lives per game
+- Fun to play :)
 
 ## Controls
-![Controls](./assets/images/controls.png)
- - Up, Left, Right - moving
- - Space - fire
+|              | Button              |
+|--------------|---------------------|
+| Move         | <kbd>Up</kbd>       |
+| Turn Left    | <kbd>Left</kbd>     |
+| Turn Right   | <kbd>Right</kbd>    |
+| Fire bullets | <kbd>Spacebar</kbd> |
+
 
 ## How to play
-Go to [this link](https://petrischakmaxim.github.io/space-shooter/)
+- Online:
+  - Go to [this link](https://petrischakmaxim.github.io/space-shooter/)
+- Your PC:
+  - Download this repository
+  - Install [Node JS](https://nodejs.org/en/download)
```

Comment:
This will be used by developers only, so it's safe to presume that Node.js is already installed. Consider replacing 
 **- Install [Node JS](https://nodejs.org/en/download)
 - Enter `npm i`**
 with single entry
 **- install dependencies (`npm i`)**